src/pages/ThorChainLP/Position/Position.tsx

```diff
@@ -49,36 +48,35 @@ import { PoolInfo } from '../components/PoolInfo'
 import { RemoveLiquidity } from '../components/RemoveLiquidity/RemoveLiquidity'
 import { usePools } from '../queries/hooks/usePools'
 import { useUserLpData } from '../queries/hooks/useUserLpData'
+import { getPositionName } from '../utils'
 
 type MatchParams = {
-  poolAccountId?: AccountId
-  poolOpportunityId?: string
+  poolAssetId: string
+  accountId: AccountId
+  opportunityId: string
 }
 
 const containerPadding = { base: 6, '2xl': 8 }
 const maxWidth = { base: '100%', md: '450px' }
 const responsiveFlex = { base: 'auto', lg: 1 }
 
 const PoolHeader = () => {
-  const translate = useTranslate()
   const history = useHistory()
-  const { path } = useRouteMatch()
-  const handleBack = useCallback(() => {
-    const isPoolPage = matchPath('/pools/positions/:poolAssetId', path)
-    const isPoolAccountPage = matchPath('/pools/poolAccount/:poolAccountId/:poolAssetId', path)
-
-    if (isPoolAccountPage) {
-      history.push('/pools/positions')
-    } else if (isPoolPage) {
-      history.push('/pools')
-    }
-  }, [history, path])
+  const translate = useTranslate()
+
+  const handleBack = useCallback(() => history.push('/pools/positions'), [history])
```

Comment:
just go back to positions

Reply:
Indeed - now that we have separate position and pool pages, this makes absolute sense